src/main/scala/com/typesafe/sbt/packager/rpm/RpmPlugin.scala

```diff
@@ -81,7 +81,7 @@ object RpmPlugin extends AutoPlugin {
     rpmConflicts := Seq.empty,
     rpmSetarch := None,
     rpmChangelogFile := None,
-    rpmBrpJavaRepackJars := true,
+    rpmBrpJavaRepackJars := false,
```

Comment:
The documentation regarding [jar repackaging](http://www.scala-sbt.org/sbt-native-packager/formats/rpm.html#jar-repackaging) is now going to be outdated (and generally it's pointing to a blog post that no longer exists). Where is the documentation maintained?